StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetLoader.swift

```diff
@@ -189,6 +190,51 @@ final class PaymentSheetLoader {
         return intent
     }
 
+    static func additionalParams(configuration: PaymentSheet.Configuration) -> [String: Any]? {
+        if let ephemeralKey = configuration.customer?.ephemeralKeySecret {
+            return ["legacy_customer_ephemeral_key": ephemeralKey]
+        }
+        return nil
+    }
+
+    static func getPaymentMethodsStrategy(intent: Intent, configuration: PaymentSheet.Configuration) -> Task<[STPPaymentMethod], Error> {
```

Comment:
Naming nit: I'd expect this to return a "strategy" given its name

Why return a Task vs. making it like `static func getPaymentMethods(..) async throws -> [STPPaymentMethod]`?

Reply:
No good reason, I think i tried doing something like this, but ended up returning tasks. Updated, thanks!